src/ui_contextmenu.cpp

```diff
@@ -293,7 +300,79 @@ class OpenLyricsContextSubItem : public contextmenu_item_simple
                                                  core_api::get_main_window(),
                                                  "Marking tracks as instrumental...");
             } break;
+            case cmd_seek_to_next_lyric: {

```

Comment:
Braces go on their own line. I really need to setup clang-tidy so that this is enforced everywhere (including in my own code!) and flagged automatically =/

Reply:
Actually ignore me here. I'll add tooling and it'll be done there.